### DISCUSSION TITLE
Question about setting resource limits

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17739] [Q&A] [answered]
Hi, in the Helm chart, what is the difference between setting resource limits on the `runK8sConfig.containerConfig` or directly in `resources`? I’m not sure what pod main’s container refers to

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C014N0PK37E/p1699301026557099?thread_ts=1699301026.557099&cid=C014N0PK37E).*

**Comment by gibsondan** [comment, marked as answer]
These two fields will have the same effect - the top-level resources is just a short-cut for setting resources specifically, but runK8sConfig.containerConfig lets you set arbitrary configuration on the container.
